### PR TITLE
Add a sample component to the root Storybook (component from another package)

### PR DESCRIPTION
**Pull request by okmttdhr**
Related to https://github.com/Automattic/wp-calypso/issues/75167, https://github.com/Automattic/wp-calypso/pull/75942

## Proposed Changes

* Add a story for `ThemeCard` in `packages/design-picker` (c.f. https://github.com/Automattic/wp-calypso/pull/75422).
	* This can be a sample story for components outside of `client/*`, which can be helpful when we migrate `@automattic/components` into the root Storybook.
* Add `addon-viewport`.
	* I was trying to add [`addon-essentials`](https://storybook.js.org/docs/react/essentials/introduction), but it was not that easy.

## Testing Instructions

* Run `yarn storybook:start` locally
* Access to http://localhost:6006/?path=/story/packages-design-picker-themecard--not-active

https://user-images.githubusercontent.com/5287479/232993246-5587a343-0bf5-4513-96e3-cec0677e9caa.mov

For newly added addon;

https://user-images.githubusercontent.com/5287479/233285032-23fc19d5-4984-4ca2-b572-270b1af5a0b1.mov

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -2,4 +2,64 @@ import './index.scss';
 
 export const parameters = {
 	layout: 'fullscreen',
+	viewport: {
+		viewports: {
+			HUGE: {
```
Comment (okmttdhr) [inline .storybook/preview.js:23]: These viewports is from Gutenberg. https://github.com/WordPress/gutenberg/blob/db2f753977d1b5c28f1ff80b3fc32aa1d7a6a773/packages/base-styles/_breakpoints.scss#L6-L13

**Review by okmttdhr — COMMENTED** [review]
Looking into failing tests 🙄

**Comment by noahtallen** [comment]
One note is that the components storybook is extremely simple: https://github.com/Automattic/wp-calypso/blob/9fb7ac149e55361c0c3a9639ad0caa5e15fc88b0/packages/components/.storybook/main.js#L3

It might be worth just seeing what happens when importing all of components! I think they would just need to go under a `@automattic/components` category

**Comment by okmttdhr** [comment]
> It might be worth just seeing what happens when importing all of components! I think they would just need to go under a `@automattic/components` category

I created another PR, and it just worked! https://github.com/Automattic/wp-calypso/pull/76007
Please take a look if you have time. 🙂 @noahtallen

File: .storybook/preview.js (hunk 1 above)
Comment (tyxla) [reply .storybook/preview.js:23]: I'd suggest keeping a comment here to reference that for future updates.

File: packages/design-picker/src/components/theme-card/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+} as Meta;
+
+type ThemeCardStory = Story< ComponentProps< typeof ThemeCard > >;
+const Template: ThemeCardStory = ( args ) => <ThemeCard { ...args } />;
+
+const Image = () => (
+	<img
+		alt="Stewart is a modern blogging theme with a left sidebar. Its default color scheme is a striking combination of orange and light gray, to give your blog a sophisticated appearance from day one."
+		src="https://i0.wp.com/s2.wp.com/wp-content/themes/pub/stewart/screenshot.png"
+	></img>
```
Comment (tyxla) [inline packages/design-picker/src/components/theme-card/index.stories.tsx:22]: We could use a self-closing component syntax:
```suggestion
	/>
```

File: packages/design-picker/src/components/theme-card/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+);
+const defaultArgs = {
+	name: 'Stewart',
+	image: <Image></Image>,
```
Comment (tyxla) [inline packages/design-picker/src/components/theme-card/index.stories.tsx:26]:
```suggestion
	image: <Image />,
```

**Review by tyxla — APPROVED** [review]
LGTM 👍 🚀

File: .storybook/preview.js (hunk 1 above)
Comment (okmttdhr) [reply .storybook/preview.js:23]: Addressed in https://github.com/Automattic/wp-calypso/pull/75943/commits/1556b5026cd3e96bdeab0559ad6486796e9d0306

**Pull request merged**